src/main/java/com/mabl/integration/jenkins/domain/CreateDeploymentPayload.java

```diff
@@ -1,18 +1,22 @@
 package com.mabl.integration.jenkins.domain;
 
-import java.util.Set;
+import org.springframework.util.StringUtils;
+
+import java.util.Arrays;
+import java.util.Collection;
 
 public class CreateDeploymentPayload {
     final String environmentId;
     final String applicationId;
-    final Set<String> planLabels;
+    final Collection<String> planLabels;
     final String sourceControlTag;
     final CreateDeploymentProperties properties;
 
-    public CreateDeploymentPayload(String environmentId, String applicationId, Set<String> planLabels, String mablBranch, CreateDeploymentProperties properties) {
+    public CreateDeploymentPayload(String environmentId, String applicationId, String planLabels, String mablBranch, CreateDeploymentProperties properties) {
         this.environmentId = environmentId;
         this.applicationId = applicationId;
-        this.planLabels = planLabels == null || planLabels.isEmpty() ? null : planLabels;
+        this.planLabels = planLabels == null || planLabels.length() == 0 ? null :
```

Comment:
Minor: I think `StringUtils.isBlank()` does the same thing as this conditional.

Reply:
Good point. I just realized that here I am using the StringUtils from Spring and not the StringUtils from Apache Commons. Nothing that a static import could not solve ;-)